### PR TITLE
feature: unified integration tests

### PR DESCRIPTION
**Pull request by samb-local**
LOCAL-746 - Integration tests for invalid trade state changes

### CODE REVIEW

**Comment by davirds** [comment]
@samb-local can you resolve the conflict?

**Comment by samb-local** [comment]
> @samb-local can you resolve the conflict?

done.

File: app/.env.sample [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CONTRACTS=../contracts/artifacts
```
Comment (davirds) [inline app/.env.sample:1]: The `app/.env.sample` is a great standard for documenting the variables you need to run the project. :D 

Can you create something similar in `deploy/`?

**Review by davirds — CHANGES_REQUESTED** [review]
Hey Sam, great Pull Request, with lots of good changes and everything seems to be working well!  

I left some comments, nothing extreme, just some suggestions and improvements

File: app/.env.sample (hunk 1 above)
Comment (samb-local) [reply app/.env.sample:1]: Great idea, I actually merged both .env files, now we just need one inside the `app` folder.

**Comment by samb-local** [comment]
@davirds comments addressed, thank you for the thorough review!

**Review by davirds — APPROVED** [review]
LGTM!

**Pull request merged**